packages/client/src/components/image/Image.tsx

```diff
@@ -0,0 +1,20 @@
+import { ImageBody, ImageWrapper } from './Image.styled';
+
+export interface ImageProps {
+  src: string;
+  alt: string;
+  height?: string;
+  width?: string;
+  borderRadius?: string;
+}
```

Comment:
Unnecessary type for img, what if in the future you will need some of the img default attributes? You will need to think about what type has every attribute of img element.
You can use built-in img type `React.DetailedHTMLProps<React.ImgHTMLAttributes<HTMLImageElement>, HTMLImageElement>`.